docs/source/api/diffusion_pipeline.mdx

```diff
@@ -30,11 +30,6 @@ Any pipeline object can be saved locally with [`~DiffusionPipeline.save_pretrain
 
 ## DiffusionPipeline
 [[autodoc]] DiffusionPipeline
-	- from_pretrained
```

Comment:
@yiyixuxu @williamberman @patil-suraj @pcuenca @anton-l Let's make sure to never add any methods here actually because it means that **only** those methods are shown. If we don't add any methods, we simply show **all** public methods by default which is much better IMO.